platform/dockeree/aif-uninstall.sh

```diff
@@ -0,0 +1,131 @@
+echo "Starting AIFabric De-Provisioning... Please ignore some of the 'No Resources found' errors, as they are harmless."
+
+echo "Instlling Helm on the machine"
+curl -L https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash -s -- --version v3.1.3
+
+echo "Uninstalling AIFabric Services"
+helm delete oob-scheduler-crd
+
+helm delete oob-installer-crd
+
+helm -n aifabric delete cronjob
+
+helm -n aifabric delete ai-app
+
+helm -n aifabric delete ai-appmanager
+
+helm -n aifabric delete ai-trainer
+
+helm -n aifabric delete ai-deployer
+
+helm -n aifabric delete ai-pkgmanager
+
+helm -n aifabric delete ai-helper
+
+helm -n aifabric delete rabbitmq
+kubectl -n aifabric patch pvc data-rabbitmq-0 -p '{"metadata":{"finalizers":null}}'
+kubectl -n aifabric delete pvc data-rabbitmq-0
+kubectl -n aifabric patch pvc data-rabbitmq-1 -p '{"metadata":{"finalizers":null}}'
+kubectl -n aifabric delete pvc data-rabbitmq-1
+kubectl -n aifabric patch pvc data-rabbitmq-2 -p '{"metadata":{"finalizers":null}}'
+kubectl -n aifabric delete pvc data-rabbitmq-2
+
+helm -n aifabric delete jwksproxy
+
+# configmaps and secrets will be deleted with delete namespace
+echo "Deleting AIFabric Namespace"
+kubectl delete ns aifabric
+
+# EFK Uninstall
+echo "Uninstalling EFK Stack"
+helm delete efk-stack-crd
+
+# Istio uninstall
+echo "Uninstalling Istio from Cluster"
+curl -sL "https://github.com/istio/istio/releases/download/1.5.6/istio-1.5.6-linux.tar.gz" | tar xz
+cd istio-1.5.6
+chmod +x ./bin/istioctl
+./bin/istioctl manifest generate | kubectl delete --ignore-not-found=true -f -
+kubectl delete namespace istio-system
+
+# Kots-Admin Uninstall
+echo "Unistalling Kots-Admin from the Cluster"
+kubectl delete ns aif-core
+
+# Ceph uninstall
+echo "Uninstalling Ceph from Cluster"
+kubectl delete -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/block_storage_class.yaml
+
+kubectl delete -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/block_storage.yaml
+
+kubectl delete -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/object_storage_class.yaml
+
+kubectl delete -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/object.yaml
+
+kubectl delete -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/cluster-test.yaml
+
+kubectl delete -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/operator.yaml
```

Comment:
will it throw any error in case the install YAML file is different from this current one?

Reply:
Yes @vineetsingh5 not the whole content but if metadat name is different it will not delete the actual instance, but script will stilll continue, right now it is just safe guarded by rook-ceph namespace deletion altogether. I have created this task https://uipath.atlassian.net/browse/AIFBR-6730 to make it use our scripts from oneinstaller.